### PR TITLE
Fix for H1 Covered by Codefund ads.

### PR DESCRIPTION
**Pull request by GeorgeWL**
Thanks for taking the effort to create a PR!

If you are creating an extensive PR, you might want to open an issue with your idea first, so that you don't put a lot of effort in an PR that wouldn't be accepted. Please prepend pull requests with `WIP: ` if they are not yet finished
PR checklist:

* [ ] Added unit tests
* [ ] Updated changelog
* [x] Updated docs (either in the description of this PR as markdown, or as separate PR on the `gh-pages` branch. Please refer to this PR). For new functionality, at least [API.md](https://github.com/mobxjs/mobx/blob/gh-pages/docs/refguide/api.md) should be updated
* [ ] Added typescript typings
* [ ] Verified that there is no significant performance drop (`npm run perf`)

Feel free to ask help with any of these boxes!

The above process doesn't apply to doc updates etc.

### CODE REVIEW

**Comment by GeorgeWL** [comment]
#2027

File: docs/book.json [hunk 1]
```diff
@@ -14,5 +14,12 @@
         "disqus": {
             "shortName": "mobservable"
         }
+    },
+    "styles": {
+        "website": "styles/website.css",
```
Comment (FredyC) [inline docs/book.json]: I see only this css file in that folder. What are those others for?

**Comment by FredyC** [comment]
@GeorgeWL Can you please fix that small nitpick so we can merge it?

File: docs/book.json (hunk 1 above)
Comment (GeorgeWL) [reply docs/book.json]: tbh, wasn't sure if it needed them to be defined, I'll drop that part so it's ready for merge

**Comment by GeorgeWL** [comment]
@FredyC fixed it 👍

**Pull request merged**